nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/test/java/org/nuxeo/ecm/platform/tag/AbstractTestTagService.java

```diff
@@ -890,6 +905,27 @@ public void testTagDoesNotChangeLastContributor() {
                                .size());
     }
 
+    /**
+     * The {@link TaggedVersionListener} when asynch, should handle removed already documents.
+     *
+     * @since 11.1
+     */
+    @Test
+    @LogCaptureFeature.FilterOn(logLevel = "ERROR")
+    public void testTaggedVersionListenerHandleRemovedDocuments() {
+        DocumentModel doc = session.createDocumentModel("/", "doc", "File");
+        doc = session.createDocument(doc);
+        session.save();
```

Comment:
Not sure this is strictly necessary, @efge?

Reply:
seems to be working without saving indeed.